mito-ai/src/Extensions/ErrorMimeRenderer/OutputBlock.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState, useEffect } from 'react';
+
+interface OutputBlockProps {
+    message: string;
+}
+
+export const OutputBlock = ({ message }: OutputBlockProps) => {
```

Comment:
Maybe we can call this `CollapsibleWarningBlock` since it can only be used with warnings right now